### PR TITLE
Mobile menu bar

### PR DESCRIPTION
**Pull request by AlphaNtihinduka**
The mobile menu bar was added.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -325,5 +325,15 @@ <h6 class="languages-world">Languages</h6>
...
       </div>
     </footer>
+
+    <div class="menu-bar">
+         <i class="fa fa-times"></i>
+        <ul class="header-slide-links">
+          <li><a href="#">Portifolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
```
Comment (gemmen29) [inline index.html]: - [ ] Kindly make when the user press any of the mobile menu options, a correct part of the page is displayed. then, the menu closed.

File: index.html [hunk 2]
```diff
@@ -325,5 +325,15 @@ <h6 class="languages-world">Languages</h6>
...
+    </div>
```
Comment (gemmen29) [inline index.html]: - [ ] Kindly add the blur effect to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @AlphaNtihinduka, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @AlphaNtihinduka, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**